performance.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Plugin Name: Performance Lab Monorepo (not a real plugin)
+ * Plugin URI: https://github.com/WordPress/performance
+ * Description: The Performance Monorepo is not a plugin rather a collection of performance features as plugins. Download <a href="https://wordpress.org/plugins/performance-lab/" to install performance features instead.
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * Text Domain: performance-lab
```

Comment:
I like the idea of providing some feedback to anyone who tries to install the whole repo. I think using the `performance-lab` text domain is likely fine. I wonder if we could even automatically load the root file for the actual Performance Lab plugin if someone ends up in this situation?

Reply:
@joemcgill For users utilizing wp-env, it should function as expected. However, for those with custom setups, we should include documentation on how to work with the monorepo. Simply activating the PL plugin isn't sufficient since we need to provide references to other plugins as well. It could also be confusing if one plugin from the `/plugins` directory is loaded while others are not.

@westonruter Any thoughts here?

Reply:
We're currently lacking such documentation for users who want to work on the standalone plugins as well, so I don't see this as a blocker.

Reply:
I've updated notices in 8cbac0c1f2ae3ed6d8c943192b85b1a1457d7b39 with a link to the handbook page. We can add documentation there for how to handle contributing when not using wp-env.

Reply:
![image](https://github.com/WordPress/performance/assets/134745/a3f1d6aa-be3b-4160-8807-b402517e599c)

Reply:
I think this is a good step. Thanks.